### ISSUE TITLE
About the fuction of resizing data durning loss calculation

### ISSUE THREAD

**Issue by wenmingMeng** [jianfenglihg/UnOpticalFlow#2]
Thanks for your sharing, i have some questions。
https://github.com/jianfenglihg/UnOpticalFlow/blob/3caa4d6f7f472da07a96d9d452546893c0eced3c/loss_functions.py#L45
why use AdaptiveAvgPooling2D here？
And the author of github.com/ClementPinard/SfMLearner-Pytorch use F.interpolate here.
Is AdaptiveAvgPooling2D better than F.interpolate ?
Looking forwrd to your reply。
Thanks for your sharing againg！

Appending：
And i found that different resizing fuction  will cause different loss value during training。I don't know whether it will cause a positive or negative impact on the performance of model。

**Comment by jianfenglihg** [comment]
avg_pool resizing  inherited from github.com/anuragranj/cc, i think avg-pool is equivalent to mean filtering, you can think of it as a way of image preprocessing to filter noise. I have not verified the effectiveness of this.

**Comment by wenmingMeng** [comment]
Thanks for your reply！
